pkg/operation/botanist/logging.go

```diff
@@ -74,5 +74,13 @@ func (b *Botanist) DeploySeedLogging(ctx context.Context) error {
 		}
 	}
 
+	// TODO (ialidzhikov): remove in a future version
+	stsKeys := []client.ObjectKey{
+		kutil.Key(b.Shoot.SeedNamespace, "loki"),
+	}
+	if err := common.DeleteStatefulSetsHavingDeprecatedRoleLabelKey(ctx, b.K8sSeedClient.Client(), stsKeys); err != nil {
```

Comment:
Aren't we required to wait for the STS to be gone before continuing?

Reply:
I reverted this STS migration logic for now. I will tackle it in another PR as suggested initially.